tests/test_fb_copy_ad.R

```diff
@@ -0,0 +1,339 @@
+library(testthat)
```

Comment:
Although I appreciate the effort to test these, but without a token and actual ids (and names), this approach will not really work -- so I suggest removing all these test files from the PR.